mapproxy/cache/file.py

```diff
@@ -45,14 +47,15 @@ def __init__(self, cache_dir, file_ext, directory_layout='tc',
         self.link_single_color_images = link_single_color_images
         self._tile_location, self._level_location = path.location_funcs(layout=directory_layout)
         if self._level_location is None:
-            self.level_location = None # disable level based clean-ups
+            self.level_location = None  # disable level based clean-ups
 
     def tile_location(self, tile, create_dir=False, dimensions=None):
-        if dimensions is not None and len(dimensions)>0:
+        if dimensions is not None and len(dimensions) > 0:
             items = list(dimensions.keys())
             items.sort()
             dimensions_str = ['{key}-{value}'.format(key=i, value=dimensions[i].replace('/', '_')) for i in items]
-            cache_dir = os.path.join(self.cache_dir, '_'.join(dimensions_str))
+            # todo: cache_dir is not used. should it get returned or removed?
+            cache_dir = os.path.join(self.cache_dir, '_'.join(dimensions_str))  # noqa
```

Comment:
This might be an error that existed before. This could get adressed in a future PR

Reply:
There also already exists an issue for